src/WebJobs.Extensions/Extensions/Warmup/Trigger/WarmupTriggerBinding.cs

```diff
@@ -0,0 +1,57 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs.Host.Bindings;
+using Microsoft.Azure.WebJobs.Host.Listeners;
+using Microsoft.Azure.WebJobs.Host.Protocols;
+using Microsoft.Azure.WebJobs.Host.Triggers;
+
+namespace Microsoft.Azure.WebJobs.Extensions.Warmup.Trigger
+{
+    internal class WarmupTriggerBinding : ITriggerBinding
+    {
+        private readonly ParameterInfo _parameter;
+
+        public WarmupTriggerBinding(ParameterInfo parameter)
+        {
+            _parameter = parameter;
+        }
+
+        public Type TriggerValueType
+        {
+            get { return _parameter.ParameterType; }
```

Comment:
With the new Type, this should be return typeof(WarmupContext) - similar to TimerTrigger